lib/3scale_toolbox/helper.rb

```diff
@@ -1,24 +1,36 @@
 module ThreeScaleToolbox
-  module Helper
-    def self.compare_hashes(first, second, keys)
-      keys.map { |key| first.fetch(key, nil) } == keys.map { |key| second.fetch(key, nil) }
-    end
+  class Helper
```

Comment:
Why is it a class? Do you plan make instance of it?

Reply:
No, but I did not plan to include it either. 

Changing to `module`